packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -271,16 +300,24 @@ export class SerializedStateManager {
  */
 export async function getLatestSnapshotInfo(
 	mc: MonitoringContext,
-	storageAdapter: Pick<
-		IDocumentStorageService,
-		"getSnapshot" | "getSnapshotTree" | "getVersions" | "readBlob"
-	>,
+	storageAdapter: ISerializedStateManagerDocumentStorageService,
 	supportGetSnapshotApi: boolean,
 ): Promise<ISnapshotInfo | undefined> {
 	return PerformanceEvent.timedExecAsync(
 		mc.logger,
 		{ eventName: "GetLatestSnapshotInfo" },
 		async () => {
+			if (supportGetSnapshotApi && storageAdapter.loadedGroupIdSnapshots !== undefined) {
+				const snapshot = await storageAdapter.getSnapshot?.({
+					versionId: undefined,
+					scenarioName: "getLatestSnapshotInfo",
+					cacheSnapshot: false,
+					loadingGroupIds: Object.keys(storageAdapter.loadedGroupIdSnapshots).concat(""),
```

Comment:
@tyler-cai-microsoft Is here the place where we want to fetch the ungrouped/default snapshot along with the other grouped snapshot?

Reply:
We just want to fetch the groupId snapshots here. The default snapshot is already downloaded.